examples/arkane/networks/CH2NH2/yaml_files/CH2NH.yml

```diff
@@ -1,245 +0,0 @@
-RMG_version: 3.0.0
```

Comment:
What were these example files for previously? Why remove it?

Reply:
Ultimately added these back, see discussion [here](https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2504#discussion_r1273873059) about cleaning up the history